.travis.yml

```diff
@@ -15,6 +15,6 @@ script: make test
 env:
     - IDEA_VERSION=13.1.6
     - IDEA_VERSION=14.0.4
-    - IDEA_VERSION=14.1.1
+    - IDEA_VERSION=14.1.4
```

Comment:
I was occupied with a P1/S1 bug, and tried to review after travis builds pass - assumed that some unit test is failing
sorry that I didn't pay attention to why it was failing